charts/aws-ebs-csi-driver/templates/node-windows.yaml

```diff
@@ -62,26 +65,28 @@ spec:
           env:
             - name: CSI_ENDPOINT
               value: unix:/csi/csi.sock
-{{- if .Values.proxy.http_proxy }}
-            - name: HTTP_PROXY
-              value: {{ .Values.proxy.http_proxy | quote }}
-            - name: HTTPS_PROXY
-              value: {{ .Values.proxy.http_proxy | quote }}
-            - name: NO_PROXY
-              value: {{ .Values.proxy.no_proxy | quote }}
-{{- end }}
+            - name: CSI_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            {{- if .Values.proxy.http_proxy }}
+            {{- include "aws-ebs-csi-driver.http-proxy" . | nindent 12 }}
+            {{- end }}
+            {{- with .Values.node.env }}
+            {{- . | toYaml | nindent 12 }}
+            {{- end }}
           volumeMounts:
             - name: kubelet-dir
               mountPath: C:\var\lib\kubelet
               mountPropagation: "None"
             - name: plugin-dir
               mountPath: C:\csi
             - name: csi-proxy-disk-pipe
-              mountPath: \\.\pipe\csi-proxy-disk-v1beta2
+              mountPath: \\.\pipe\csi-proxy-disk-v1
```

Comment:
Why does the mount path have a version in it?

Reply:
that is the way csi-proxy works, it drops a socket at a path for each api it serves